tests/test_tracebacks.py

```diff
@@ -732,12 +734,72 @@ def test_json_traceback_value_error(
         tracebacks.ExceptionDictTransformer(**kwargs)
 
 
-def test_exception_dict_transformer_missing_exc_info():
+class TestLogException:
     """
-    ExceptionDictTransformer returns an empty list if exc_info is missing.
+    Higher level "integration tests" for "Logger.exception()".
     """
-    transformer = tracebacks.ExceptionDictTransformer()
 
-    result = transformer(exc_info=(None, None, None))
+    @pytest.fixture
+    def cap_logs(self) -> structlog.testing.LogCapture:
```

Comment:
you can make these static since you don't use self

Reply:
No. This breaks pytest on older Python versions (`AttributeError: 'staticmethod' object has no attribute '__name__'`) and fixing _that_ is more cumbersome than living with a non-static method not using self. :-)